identity-and-access-management/key-manager/concepts.mdx

```diff
@@ -0,0 +1,152 @@
+---
+meta:
+  title: Key Manager - Concepts
+  description: Explore essential cryptographic concepts, including symmetric and asymmetric encryption, data encryption keys (DEKs), key encryption keys (KEKs), and Scaleway Key Manager's robust features for secure key management and encryption operations.
+content:
+  h1: Concepts
+  paragraph: Explore essential cryptographic concepts, including symmetric and asymmetric encryption, data encryption keys (DEKs), key encryption keys (KEKs), and Scaleway Key Manager's robust features for secure key management and encryption operations.
+tags: key-manager key encryption-key
+dates:
+  validation: 2025-01-06
+---
+
+## Asymmetric encryption
+
+Asymmetric encryption is a fundamental type of cryptographic method used to secure data with a pair of keys: a **public key** and a **private key**.
+
+The public key is used for encryption and can be shared openly, while the private key is used for decryption and must be kept secret. This design eliminates the need to share a single key securely, which is a challenge in symmetric encryption.
+
+Asymmetric encryption is particularly well-suited for secure communication and authentication, such as encrypting emails or verifying digital signatures. However, it is slower than symmetric encryption. Algorithms like RSA and ECC are common examples of asymmetric encryption.
+
+## Ciphertext
+
+Ciphertext refers to data that has been encrypted using a cryptographic algorithm and a key.
+
+Unlike [plaintext](#plaintext), ciphertext is not human-readable and cannot be understood or used without first decrypting it with the appropriate decryption key.
```

Comment:
I would like to add that ciphertext can be encrypted on client side, as long as the encryption key used to encrypt it is safely stored, for example in Key Manager